DataFormats/SiStripCluster/interface/SiStripCluster.h

```diff
@@ -15,28 +15,28 @@ class SiStripCluster  {
   static const uint16_t stripIndexMask = 0x7FFF;  // The first strip index is in the low 15 bits of firstStrip_
   static const uint16_t mergedValueMask = 0x8000;  // The merged state is given by the high bit of firstStrip_
 
+
   /** Construct from a range of digis that form a cluster and from 
    *  a DetID. The range is assumed to be non-empty.
    */
   
-  SiStripCluster() : error_x( -99999.9 ) {}
+  SiStripCluster() {}
 
   explicit SiStripCluster(const SiStripDigiRange& range);
 
   template<typename Iter>
   SiStripCluster(const uint16_t& firstStrip, 
 		 Iter begin, Iter end ):
-	 amplitudes_(begin,end), firstStrip_(firstStrip), 
+	 amplitudes_(begin,end), firstStrip_(firstStrip) {}
   // ggiurgiu@fnal.gov, 01/05/12
   // Initialize the split cluster errors to un-physical values.
```

Comment:
This comment should be moved close to L95, where there is now the actual initialization of the cluster errors to un-physical values

Reply:
@perrotta, PR is updated